packages/components/README.md

```diff
@@ -6,56 +6,67 @@ Reusable UI components for Deriv.
 
 **In this document**
 
--   [Other documents](#other-documents)
 -   [Commands](#commands)
 -   [Storybook](#storybook)
 -   [Usage](#usage)
 
-## Other documents
-
--   [General](docs/README.md) - Contains general philosophy and overview of this package
--   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding, code usage)
-
 ## Commands
 
 Setup:
+The `npm run bootstrap` command should be run in the root of the project, because it does not exist in the components package.js
 
 ```sh
-lerna bootstrap
+npm run bootstrap
 ```
 
 Serve:
+The `npm run serve` command is used to serve the components package
 
 ```sh
 npm run serve
 ```
 
 Build:
+The `npm run build` command is used to build the components package
```

Comment:
Maybe it is better to add this before serve command , because at first we run `npm run bootstrap` then `npm run build`